index.html

```
@@ -1,87 +1,105 @@
-					<p>Berkshire College of Agriculture | 2012 - 2014</p>
-					<ul>
-						<li>Deployed and maintained desktops, laptops, printers.</li>
-						<li>Managed software deployment via Symantec Ghost and GPO.</li>
-						<li>Supported users remotely with Dameware, managed Active Directory.</li>
-					</ul>
-				</section>
-			</article>
-		</main>
-	</body>
-</html>
+<html lang="en">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)